recipe_scrapers/goodfooddiscoveries.py

```diff
@@ -0,0 +1,46 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class GoodFoodDiscoveries(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "goodfooddiscoveries.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def prep_time(self):
+        return self.schema.prep_time()
+
+    def cook_time(self):
+        return self.schema.cook_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        # removing "u003cbu003e" and "u003c/bu003e" as it looks to be used at times to bold text on the site
```

Comment:
Yeesh.  I bet that's not the only weird HTML encoding issue we'll see from this site, but nice work spotting it.  If it begins happening very frequently then we can figure out a more robust solution; string replacement seems good for now :+1:

Reply:
@jayaddison Agreed. That sounds like a good plan for moving forward for now though! 👍